src/segment-loader.js

```diff
@@ -1326,22 +1326,22 @@ export default class SegmentLoader extends videojs.EventTarget {
    * @return {boolean} True if the request was aborted, false otherwise
    * @private
    */
-  abortRequestEarly_(stats) {
+  earlyAbortWhenNeeded_(stats) {
```

Comment:
Since this function no longer aborts, the name needed a slight change. Now we only trigger `earlyabort` whenever we think that an `earlyabort` should happen and let master playlist controller handle a switch if it thinks that it should happen.